### PR TITLE
Update online_monitor for lone hits

### PR DESCRIPTION
**Pull request by jorana**
**What is the problem / what does the code in this PR do**
Use selection as proposed here to be displayed in the online monitor:
https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:zihao:lone_hits_electrode_test

**Can you briefly describe how it works?**
Apply cuts at the event-builder prior to shipping data off to mongo.

### CODE REVIEW

**Review by WenzDaniel — APPROVED** [review]
Technically it looks fine.

**Comment by jorana** [comment]
Perhaps it's good if I stress that this PR is mostly an invitation for @xzh19980906 to help thinking about what would make sense to do in the online monitor.

**Comment by xzh19980906** [comment]
To me it looks great

File: straxen/plugins/online_monitor.py [hunk 1]
```diff
@@ -49,6 +49,18 @@
...
+    strax.Option(
+        'lone_hits_cut_string',
+        type=str,
+        default='(area > 0.2) & (area < 2.0)',
+        help='Selection (like selection_str) applied to data for '
+             '"lone-hits", cuts should be separated using "&")'),
+    strax.Option(
+        'lone_hits_min_gap',
+        type=int,
+        default=15_000,
```
Comment (feigaodm) [inline straxen/plugins/online_monitor.py:61]: I didn't know that strax can identify 15_000 as a number. nice!

Comment (WenzDaniel) [reply straxen/plugins/online_monitor.py:61]: :D The credit goes to python though. It is a vanilla feature.

**Review by feigaodm — APPROVED** [review]

**Pull request merged**